### ISSUE TITLE
clickable divs are inaccessible as written

### ISSUE THREAD

**Issue by adamint** [dotnet/aspire#1392]
All clickable divs either should be a button or else have proper role and label. However, justification should be required when avoiding using a button, as this is the preferred solution.
